src/Pangraph.hs

```diff
@@ -97,27 +97,25 @@ verifyGraph vs = mapMaybe (\e -> lookupEndpoints (e, edgeEndpoints e))
 makeEdge :: [Attribute] -> (Vertex, Vertex) -> Edge
 makeEdge = Edge Nothing
 
--- | Vetex constuctor
+-- | Vetex constructor
 makeVertex :: VertexID -> [Attribute] -> Vertex
 makeVertex = Vertex
 
 -- * Pangraph Getters
 
--- | Returns the Edges from a Pangraph instance
+-- | Returns the ['Edge'] from a Pangraph instance
```

Comment:
I'd prefer `Returns the list of edges...`.

Also, to make this consistent with the naming we use in Alga, I would prefer this function to be called `edgeList`.